apps/wallet/src/ui/app/pages/accounts/ExportPassphrasePage.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useMemo } from 'react';
+
+import { Navigate, useNavigate, useParams } from 'react-router-dom';
+import { VerifyPasswordModal } from '../../components/accounts/VerifyPasswordModal';
+import Loading from '../../components/loading';
+import Overlay from '../../components/overlay';
+import { useAccountSources } from '../../hooks/useAccountSources';
+import { useExportPassphraseMutation } from '../../hooks/useExportPassphraseMutation';
+import { HideShowDisplayBox } from '_components/HideShowDisplayBox';
+import Alert from '_components/alert';
+
+export function ExportPassphrasePage() {
+	const { accountSourceID } = useParams();
+	const { data: allAccountSources, isLoading } = useAccountSources();
+	const accountSource = useMemo(
+		() => allAccountSources?.find(({ id }) => id === accountSourceID) || null,
+		[allAccountSources, accountSourceID],
+	);
+	const navigate = useNavigate();
+	const exportMutation = useExportPassphraseMutation();
+
+	if (!isLoading && accountSource?.type !== 'mnemonic') {
+		return <Navigate to={'/accounts/manage'} />;
+	}
+	return (
+		<Overlay title="Export Passphrase" closeOverlay={() => navigate(-1)} showModal>
+			<Loading loading={isLoading}>
+				{exportMutation.data ? (
+					<div className="flex flex-col gap-3 min-w-0">
+						<Alert>
+							<div className="break-normal">Do not share your Passphrase!</div>
+							<div className="break-normal">
+								It provides full control of all accounts derived from it.
+							</div>
+						</Alert>
+						{<HideShowDisplayBox value={exportMutation.data} copiedMessage="Passphrase copied" />}
```

Comment:
don't need the `{``}` here

Reply:
Oh yeah took me some time to realise what you mean! I was seeing the brackets for some reason. Thanks I will update it